tests/fixtures/pulp.py

```diff
@@ -164,11 +164,22 @@ async def func(*args, **kwargs):
 @pytest.fixture
 def create_module(monkeypatch):
     async def func(*args, **kwargs):
+        _, modules_yaml, name, stream, release, arch = args
```

Comment:
I'm always getting `virt` module for `x86_64`, but in `modules_yaml` I get both `virt` and `virt-devel`. Need to do something about this